roles/kubevirt-resources/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+namespace: "kube-system"
+docker_prefix: "kubevirt"
+docker_tag: "latest"
```

Comment:
I would add default for `action` parameter as well.

Reply:
1. I think that the default should be the released version (latest is not static).

Reply:
If devs are going to use this to deploy kubevirt then I think it should be latest.